easybuild/easyconfigs/v/VTK/VTK-8.1.0-foss-2018a-Python-2.7.14.eb

```diff
@@ -44,7 +45,8 @@ dependencies = [
     ('X11', '20180131'),
 ]
 
-configopts = "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
+configopts = "-DVTK_Group_MPI:BOOL=ON "
```

Comment:
Why this particular order? I'd suggest to group by either functionality or alphabetically. Now it seems random.

Reply:
@damianam Like this it's consistent with the existing VTK 8.1.1 easyconfig, see https://github.com/easybuilders/easybuild-easyconfigs/pull/6404/files#diff-0ee45fd4c5d93b140383c1df76708f6eR48, so I prefer leaving it like this.

I have no particular reason to order the options like this though, so if you insist I'll just tweak the VTK 8.1.1 easyconfig as well to match it.

Reply:
I think it makes sense to use a sensible order, now that you are changing the easyconfig.